src/storage/mvcc/txn.rs

```diff
@@ -327,14 +336,15 @@ impl<S: Snapshot> MvccTxn<S> {
                 key: key.into_raw()?,
             });
         }
+
         // No need to check data constraint, it's resolved by pessimistic locks.
         self.put_lock(
```

Comment:
we can reduce 2 parameters for this function.

Reply:
yes